README.md

```diff
@@ -81,6 +81,8 @@ Configuration of OpenVINO for a model is done through the Parameters section of
 * `SKIP_OV_DYNAMIC_BATCHSIZE `: The topology of some models do not support openVINO dynamic batch sizes. Set the value of this parameter to `YES`, in order
 to skip the dynamic batch sizes in backend.
 * `ENABLE_BATCH_PADDING `: By default an error will be generated if backend receives a request with batch size less than max_batch_size specified in the configuration. This error can be avoided at a cost of performance by specifying `ENABLE_BATCH_PADDING` parameter as `YES`.
+* `RESHAPE_IO_LAYERS `: By setting this parameter as `YES`, the IO layers are reshaped to the dimensions provided in
+model configuration. By default, the dimensions in the model is used.
```

Comment:
Why we can't use reshape in model config for this?

Reply:
This is actually reshaping the entire OV model network in order to support the model_config specified shape. Little context.. 
The DLRM OV model has the following shape: input.1 [1,13],  lS_o [26,1] and ls_i [26,1]. For a batch size of N the network should be reshaped to [N,13], [26,N] and [26,N] respectively. The OV model reshape allows us to reshape the network to support this while loading. The mode config reshape is very restrictive and doesnot solve the problem at hand. This PR is not reshaping the tensor, but reshaping the model network to support the new dimensions.

Reply:
You are almost right. Not at the runtime, this is at the load time.

Reply:
Just out of curiosity, why the model itself doesn't specify N for batch size directly

Reply:
The users may want to try inputs with different shape than the model was created with. The PR will allow to reshape these inputs to conform with what user desires. The dynamic batch size is so restrictive in OpenVINO that the model_server in openVINOtoolkit, reloads the model reshaping the network with the upcoming request's shape.